skycoin/skywire
# Migrate update logic to Debian package

Did you run `make format && make check`? Yes

Fixes #1070 

 Changes:	
- Remove old logic, based on downloading compressed version from github release page
- Add new logic to update from Debian package repository	

How to test this PR:
- Burn fresh skybian image on SD card
- Clone this PR, and build binaries by `GOOS=linux GOARCH=arm64 GOARM=6 make build BUILDTAG=linux_arm64`. (I use `arm64` because of my board RPi4, you should use your own board architect).
- Replace SD binaries files with new built binaries. Following files should replaced in `/usr/bin`:
  - `skywire-cli`
  - `skywire-visor`
  - `setup-node`
  - `apps/*`
- Start board
- Open hypervisor UI and Click on update
- After complete and restarting service, check version of visor, should be 0.5.1.
